lib/Minz/Session.php

```diff
@@ -55,43 +59,47 @@ public static function init($name, $volatile = false) {
 
 
 	/**
-	 * Permet de récupérer une variable de session
-	 * @param string $p le paramètre à récupérer
-	 * @return mixed|false la valeur de la variable de session, false si n'existe pas
+	 * Allows you to retrieve a session variable
+	 * @param string $p the parameter to retrieve
+	 * @param mixed|false $default the default value if the parameter doesn't exist
+	 * @return mixed|false the value of the session variable, false if doesn't exist
 	 */
 	public static function param(string $p, $default = false) {
 		return $_SESSION[$p] ?? $default;
 	}
 
 
 	/**
-	 * Permet de créer ou mettre à jour une variable de session
-	 * @param string $p le paramètre à créer ou modifier
-	 * @param mixed|false $v la valeur à attribuer, false pour supprimer
+	 * Allows you to create or update a session variable
+	 * @param string $parameter the parameter to create or modify
+	 * @param mixed|false $value the value to assign, false to delete
 	 */
-	public static function _param($p, $v = false) {
+	public static function _param(string $parameter, $value = false): void {
 		if (!self::$volatile && !self::$locked) {
 			session_start();
 		}
-		if ($v === false) {
-			unset($_SESSION[$p]);
+		if ($value === false) {
+			unset($_SESSION[$parameter]);
 		} else {
-			$_SESSION[$p] = $v;
+			$_SESSION[$parameter] = $value;
 		}
 		if (!self::$volatile && !self::$locked) {
 			session_write_close();
 		}
 	}
 
-	public static function _params($keyValues) {
+	/**
+	 * @param array<string, string|bool>|array<string, array<string>|int> $keyValues
```

Comment:
```suggestion
	 * @param array<string,string|bool|int|array<string>> $keyValues
```

Light simplification, but the actual usage should be checked